bwi_logging/src/bwi_logging/logger_node.py

```diff
@@ -48,19 +48,21 @@
 from .directory import LoggingDirectory
 
 
+""" Main function. """
 def main(argv=None):
-    """ Main function. """
```

Comment:
The comment should go here, following the Python doc string convention. Various tools will find it automatically, when done that way.